CHANGELOG.md

```diff
@@ -3,6 +3,14 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [4.5.0] - TBD
+### Added
+- Variable to configure `apiary_assume_roles` cross-region S3 access.
+- Documentation in `VARIABLES.md` for the `apiary_assume_roles` variable.
+
+### Changed
+- `apiary_assume_roles[i].max_session_duration` renamed to `apiary_assume_roles[i].max_role_session_duration_seconds`.
```

Comment:
Ah, now that you put it that way should we go with a 5.0.0 version number since it's a user-facing backwards incompatible change?

Reply:
Done.